packages/appium/docs/en/guides/event-timing.md

```diff
@@ -2,4 +2,70 @@
 title: The Event Timing API
 ---
 
-TODO
+Appium comes with the ability to retrieve timing information about startup
+information and command length. This is an advanced feature that is controlled
+by the use of the `appium:eventTimings` capability (set it to `true` to log event
+timings).
+
+With this capability turned on, the `GET /session/:id` response (i.e., the
+response to `driver.getSessionDetails()` or similar, depending on client) will
+be decorated with an `events` property. This is the structure of that `events`
+property:
+
+```json
+{
+    "<event_type>": [<occurence_timestamp_1>, ...],
```

Comment:
Any reason why gitlab highlights these in red?

Reply:
it's not valid json, i guess i can just change it so that it doesn't say it's json